kuma/wiki/models.py

```diff
@@ -2037,6 +2037,12 @@ class BCSignal(models.Model):
         verbose_name=_('Document (optional)'),
         on_delete=models.SET_NULL
```

Comment:
Probably too late now but do you know what the reasoning was for this? If a document is ever deleted, what's the point of keeping a `BCSignal` row that you have no idea what browser feature it's talking about?